### DISCUSSION TITLE
Error in initializing PostgreSQL

### DISCUSSION THREAD

**Discussion by mallyapravin** [calcom/docker#251] [General]
Hi I am new to open-source contributions , I am get this error when I try to run the project via docker.

![postgres2](https://github.com/calcom/docker/assets/41922950/9f571e39-55ce-4deb-b358-f59e8df7426b)

I checked the ```.env``` file and it has the DATABASE_URL. But its not able to get the it.
![postgres1](https://github.com/calcom/docker/assets/41922950/cbc67233-223c-4e7a-83ac-b41c25a7bb15)

**Comment by krumware** [comment]
It looks like it's having trouble finding your env var in general, not necessarily having trouble with the value, via that `unexpected character "."` error

**Reply by mallyapravin** [reply]
Thanks for answering @krumware. I replaced ```ENV('DATABASE_URL')``` in schema.prisma with actual database string which seems to be working for now. But I'll try to find the cause.

**Reply by ritiksharmarj** [reply]
@mallyapravin can you share the screenshot of how you resolved this? Same issue I have.
